### ISSUE TITLE
No warnings/errors when parsing files via provided parser

### ISSUE THREAD

**Issue by qlonik** [mysticatea/vue-eslint-parser#36]
I found that when I'm using eslint, prettier, vue and typescript in Intellij IDEA, I have some weirdness with respect to warnings/errors reported by eslint and shown by IDE.

For example, when I'm using `typescript-eslint-parser` as eslint parser, I get warnings from esling+prettier as I'm typing in the TS file (`main.ts`). (There is also error about 'use latest vue-eslint-parser' which is expected and not an issue).

However, when I switch to use `vue-eslint-parser` as eslint parser, and put `typescript-eslint-parser` into `parserOptions.parser` field, then those suggestions disappear in `main.ts` file. On the other hand though, same suggestions appear in `*.vue` files. So, as I'm typing the `<script>` in `*.vue` file, all warnings/errors from eslint+prettier are reported right away.

One more, when I run `eslint --fix main.ts`, all issues are fixed correctly, so those warnings/errors are applied correctly, it is only suggestions as I'm typing in the TS source file that are not shown.

I cannot quite nail the issue here. Do you have some suggestions?

P.S. All of this is part of NativeScript project, but I think that this should not matter, since linting seems independent from NativeScript stuff.

EDIT:
From the source code of `vue-eslint-parser`, it seems to be properly calling `parseForESLint()` of whatever parser is passed (`typescript-eslint-parser` in this case) and just passing results back to eslint. So, I'm not sure why suggestions would not appear.

**Comment by mysticatea** [comment]
I apologize for my delay.

I'm not familiar with Intellij IDEA, but it sounds like the unique behavior of Intellij IDEA. I guess that the editor may check the parser setting and decide which kind of files it runs ESLint on. Please check how Intellij IDEA works.

**Comment by armano2** [comment]
Its not issue in this editor, i'm using combination eslint + vue + ts and its working fine, just make sure that your config have something like this, both of recommended configs sets parser field, and you have to override it
```
  "parser": "vue-eslint-parser",
  "parserOptions": {
      "parser": "typescript-eslint-parser",
      "sourceType": "module"
  }
```

**Comment by mysticatea** [comment]
@armano2 Thank you for the answer!

I'm closing this issue as answered.
